### PR TITLE
app-backup/zbackup: move to cmake eclass, EAPI 7 

### PR DESCRIPTION
**Pull request by SpiderX**

### CODE REVIEW

File: app-backup/zbackup/zbackup-9999.ebuild [hunk 1]
```diff
@@ -18,8 +18,8 @@ IUSE="libressl tartool"
 DEPEND="app-arch/lzma
 	dev-libs/lzo:2
 	dev-libs/protobuf:0=
-	sys-libs/libunwind:0
-	sys-libs/zlib
+	sys-libs/libunwind:=
+	sys-libs/zlib:=
```
Comment (a17r) [inline app-backup/zbackup/zbackup-9999.ebuild:22]: please don't add the slot op in this case, it was added to zlib in error.

File: app-backup/zbackup/zbackup-1.4.4-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+inherit cmake
+
+DESCRIPTION="A versatile deduplicating backup tool"
+HOMEPAGE="https://github.com/zbackup/zbackup"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+-with-openssl-exception"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
```
Comment (a17r) [inline app-backup/zbackup/zbackup-1.4.4-r2.ebuild:14]: please swap those two lines to be in line with ebuild writing standards/skel.ebuild.

**Review by a17r — CHANGES_REQUESTED** [review]

File: app-backup/zbackup/zbackup-9999.ebuild (hunk 1 above)
Comment (SpiderX) [reply app-backup/zbackup/zbackup-9999.ebuild]: Done

File: app-backup/zbackup/zbackup-1.4.4-r2.ebuild (hunk 2 above)
Comment (SpiderX) [reply app-backup/zbackup/zbackup-1.4.4-r2.ebuild]: Done

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**